health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationScreenParent/AppConfigurationWrapper.js

```
@@ -0,0 +1,349 @@
+            label={t("ADD_LOCALISATION")}
+            onClick={() => {
+              setShowPopUp(true);
+            }}
+          />
+        </SidePanel>
+      )}
+      {showPopUp && (
+        <PopUp
+          type={"default"}
+          heading={t("ADD_LOCALISATION")}
+          children={[
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)